config/default.yml

```
@@ -242,6 +242,18 @@ Gemspec/DateAssignment:
   Include:
     - '**/*.gemspec'
 
+Gemspec/DependencyVersion:
+  Description: 'Requires or forbids specifying gem dependency versions.'
+  Enabled: pending
```

There is a similar argument in #7669, I think it should be disabled by default.

```suggestion
  Enabled: false
```

---

Thanks, I think .rubocop.yml should be revert back together too, so I'll fix this directly on my local branch.